logger/src/lib.rs

```diff
@@ -282,34 +230,23 @@ pub struct Logger {
     show_level: AtomicBool,
     show_file_path: AtomicBool,
     show_line_numbers: AtomicBool,
-    level_info: LevelInfo,
-    // Used in case we want to send logs to a FIFO.
-    log_fifo: Mutex<Option<PipeLogWriter>>,
-    // Used in case we want to send metrics to a FIFO.
-    metrics_fifo: Mutex<Option<PipeLogWriter>>,
+    level: AtomicUsize,
```

Comment:
Could this be an `Atomic<Level>` ?

Reply:
Where would I get `Atomic` from? I do not find it [here](https://doc.rust-lang.org/std/sync/atomic/) and I do not want to import a new dependency like using this [crate](https://crates.io/crates/atomic).